Docs - Develop - Chains and Nodes (#1242)
iotaledger/wasp: documentation/docs/guide/chains_and_nodes/testnet.md

---
description: A public testnet for developers to try out smart contracts
image: /img/logo/WASP_logo_dark.png
keywords:
- Smart Contracts
- TestNet

---

# Testnet

The testnet is deployed for the community to use for testing and interacting with smart contracts.

> :::caution unscheduled network resets

Reviewer: I think you've been Header Capitilizing it in other PRs, should we choose one and add it to the style guide?

Author: I've added a few where appropriate, as the default labels are a bit generic.  We can certainly add it if you're in favour as well.

To make the barrier to entry for trying out the testnet as low as possible, we decided to set all possible fees to zero.
Since we expect this decision will pollute the chain quickly, we’ll perform regular, unscheduled network resets if
needed.
:::

## Introduction

This testnet is deployed with our GoShimmer cluster that backs smart contracts. Multiple committee nodes do the work for
the chain and multiple access nodes are exposed via the endpoints listed below. We do throttle the endpoints to prevent
overloading the testnet because we are looking for functionality testing more than stress testing.

## Endpoints

> You can access the testnet via a series of endpoints that have been made available.

Reviewer: ```suggestion
You can access the testnet via the following endpoints:
```
